ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1764,32 +1740,32 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-vmgenv1
-- as: azure-ipi-workers-marketimage-p2-f14
-  cron: 17 12 3,18 * *
+- as: azure-ipi-private-workers-marketimage-p2-f14
+  cron: 36 9 8,23 * *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: azure-marketplace-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       E2E_RUN_TAGS: '@azure-ipi and @network-ovnkubernetes and not @fips'
       TAG_VERSION: '@4.12'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-azure-ipi-workers-marketimage
-- as: azure-ipi-workers-marketimage-p3-f28-destructive
-  cron: 14 19 25 * *
+    workflow: cucushift-installer-rehearse-azure-ipi-private-workers-marketimage
+- as: azure-ipi-private-workers-marketimage-p3-f28-destructive
+  cron: 55 1 22 * *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: azure-marketplace-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       E2E_RUN_TAGS: '@azure-ipi and @network-ovnkubernetes and not @fips and @destructive'
       TAG_VERSION: '@4.12'
     test:
     - chain: openshift-e2e-test-qe-destructive
-    workflow: cucushift-installer-rehearse-azure-ipi-workers-marketimage
-- as: azure-ipi-workers-marketimage-p3-f28-long-duration-cloud
-  cron: 22 23 13 * *
+    workflow: cucushift-installer-rehearse-azure-ipi-private-workers-marketimage
+- as: azure-ipi-private-workers-marketimage-p3-f28-cloud
```

Comment:
May i know why `long-duration` is removed in the test name ?

Reply:
Because of https://github.com/openshift/release/pull/37510#issuecomment-1477351200

Reply:
@liangxia is it okay with current test name or need to change with another test name?

Reply:
/cc @sunzhaohua2

Reply:
Confirmed in https://github.com/openshift/release/pull/37510#issuecomment-1477385826

Reply:
Test name without long-duration while executing long-duration tests,
```
    test:
    - chain: openshift-e2e-test-qe-long-duration
```
We should remove long-duration from both place, or keep them in both place, or else we are in an inconsistent state.

Reply:
Then any suggestion for the new name of the test name?

Reply:
remove is ok to me, we know this is our team's long duration job by the suffix `-cloud` .
but wander where definds the length of the job name? can we change it? if couldn't in fact I would prefer to simplify the previous job name. maybe  `as: azure-ipi-private-workers-market-p3-f28-long-duration-cloud`?

Reply:
OK. I keep it as a backlog for myself. Let's use follow PRs to fix this kind of issues.
Merging to unblock.